weavedns/main.go

```diff
@@ -18,7 +18,8 @@ func main() {
 		justVersion bool
 		ifaceName   string
 		apiPath     string
-		dnsPort     int
+		udpPort     int
+		tcpPort     int
```

Comment:
I assumed they can be different, but if that will never be possible, I should change it... are you sure they will never be different?